src/main/resources/migrations.xml

```diff
@@ -569,9 +569,9 @@
             </column>
             <column name="task_id" type="bigint">
                 <constraints
-                    nullable="false"
-                    foreignKeyName="fk_task_parameter_task_id"
-                    references="task(id)"
+                        nullable="false"
```

Comment:
Please revert this file if there are no actual changes. Let's not give liquibase any extra possibilities to complain about changed checksums.